src/result.cpp

```diff
@@ -119,8 +127,9 @@ read_embedded_file_entry(CacheEntryReader& reader,
   uint8_t suffix_len;
   reader.read(suffix_len);
 
-  char suffix[256];
```

Comment:
When first called with the suffix `.ccmd` (from local WIP), `.ccmd` is correctly printed in `cc_log()` below because luckily `suffix` is still zero'd. The next suffix `.o` however is printed as `.ocmd` here.